mmv1/products/accesscontextmanager/ServicePerimeterDryRunEgressPolicy.yaml

```diff
@@ -0,0 +1,164 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Resource
+name: 'ServicePerimeterDryRunEgressPolicy'
+create_url: '{{perimeter}}'
+base_url: ''
```

Comment:
Why do we need an empty base_url? Would removing it have the same effect?

Reply:
Apparently it must be there or it won't build